lpvartypes_ord.pas

```diff
@@ -633,7 +633,10 @@ function TLapeType_Enum.NewGlobalVarStr(Str: UnicodeString; AName: lpString = ''
 
 function TLapeType_Enum.EvalAsSubType(Op: EOperator; Right: TLapeType): Boolean;
 const
-  BoolOperators = BinaryOperators + EnumOperators - [op_cmp_Equal, op_cmp_NotEqual];
+  BoolOperators = BinaryOperators + EnumOperators - [op_cmp_GreaterThan, op_cmp_GreaterThanOrEqual,
```

Comment:
I think this boils down to 
`BoolOperators = EnumOperators - CompareOperators - [op_Assign];`
which is equal to
`[op_Plus, op_Minus]`.

Not sure what is more clear. What do you think?